config/initializers/lograge.rb

```diff
@@ -2,11 +2,19 @@
 Rails.application.configure do
   config.lograge.enabled = true
   config.lograge.base_controller_class = 'ApplicationController'
+  # In some test specs, where we test JWT login without specifying a token,
+  # current_user raises an error, hence this fallback.
   config.lograge.custom_payload do |controller|
-    {
-      ip: controller.request.ip,
-      user_id: controller.current_user.try(:id)
-    }
+    begin
+      {
+        ip: controller.request.ip,
```

Comment:
do we really need to log IPs? Id rather not tbh

Reply:
Ok, I'll remove it.

Reply:
Fixed.